File: Python/String/Count the Number of Consistent Strings.py
class Solution:
    def countConsistentStrings(self, allowed: str, words: List[str]) -> int:
        st = set(allowed)
        cnt= 0
        for i in words:
            iscons = True
            for j in i:
                if j not in st:
                    iscons = False
                    break
            if iscons:
                cnt += 1
        return cnt
        

File: CPP/String/Count the Number of Consistent Strings.cpp
class Solution {
public:
    int countConsistentStrings(string allowed, vector<string>& words) {
        int cnt =0;
        unordered_set<char>st;
        for(auto i:allowed){
            st.insert(i);
        }
        for(auto i:words){
            string curr =i;
            bool iscons = true;
            for(auto j:curr){
                if(st.find(j)==st.end()){
                    iscons =false;
                    break;
                }
            }
            if(iscons)cnt++;
        }
        return cnt;
    }
};
